Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs

```diff
@@ -12,17 +12,15 @@
 
 namespace Mirror
 {
-
     //this script handles the functionality of the UI
-
     [InitializeOnLoad]
     public class WelcomeWindow : EditorWindow
     {
         #region Setup
 
         #region Urls
 
-        private const string WelcomePageUrl = "https://mirrorng.github.io/MirrorNG/";
+        private const string WelcomePageUrl = "https://mirrorng.github.io/MirrorNG/index.html";
         private const string QuickStartUrl = "https://mirrorng.github.io/MirrorNG/Articles/Guides/CommunityGuides/MirrorQuickStartGuide/index.html";
         private const string ChangelogUrl = "https://github.com/MirrorNG/MirrorNG/commits/master";
```

Comment:
would be nice to use https://github.com/MirrorNG/MirrorNG/blob/master/Assets/Mirror/CHANGELOG.md instead

Reply:
The UIToolkit doesn't support rich text. Do you want me to parse it so it is just plain text?